commcare_export/writers.py

```diff
@@ -519,9 +519,7 @@ def write_table(self, table):
         """
         table_name = table.name
         headings = table.headings
-        # pad the data_types to be the same length as headings so we can zip them
-        padded_data_types = table.data_types + [None] * (len(table.headings) - len(table.data_types))
-        data_type_dict = dict(zip(headings, padded_data_types))
+        data_type_dict = dict(zip_longest(headings, table.data_types))
```

Comment:
Looks like this file needs
```py
from itertools import zip_longest
```
Sorry, it was not easy to add that to my suggestion earlier.